3737/2/bot.rb

```diff
@@ -0,0 +1,15 @@
+require 'ohm'
+require 'redis'
+require 'telegram/bot'
+require_relative 'webhook_controller.rb'
+
+TOKEN = ENV['TOKEN']
```

Comment:
No need in this constant here: you just once use it here, you can pass ENV['token'] directly as a param